recipes/libcap/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
-project(PackageTest C)
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest LANGUAGES C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(PkgConfig REQUIRED)
```

Comment:
please add pkgconf to build_requirements of test_package

Reply:
done